src/v3/src/components/Link/Link.tsx

```diff
@@ -59,11 +65,10 @@ const Link: UISchemaElementComponent<{
       <LinkMui
         // eslint-disable-next-line no-script-url
         href="javascript:void(0)"
-        onClick={onClickHandler || onClick}
+        onClick={onClick}
         aria-describedby={ariaDescribedBy}
         ref={focusRef}
-        // eslint-disable-next-line react/jsx-props-no-spreading
-        {...(dataSe && { 'data-se': dataSe } )}
```

Comment:
Is it safe to do this? Appears that `dataSe` is still an optional property of `options`

Reply:
Oh nice, was worried it would be coerced into a string `'undefined'` as the value 😄